data-serving/data-service/test/controllers/case.test.ts

```diff
@@ -200,41 +207,21 @@ describe('GET', () => {
                 .expect(200);
             expect(res.body.cases).toHaveLength(1);
         });
-        it('should query results', async () => {
```

Comment:
Is there any text that the query should work over instead of notes, or is there any free-text search facility at all in the platform now? If not we can also drop the text index, which will speed up ingestion and save 13.5G in the prod cluster (the main collection is only 10.1G!)

Reply:
Agreed, in #2687